pkg/network/types.go

```diff
@@ -1,4 +1,6 @@
-package ebpf
+// +build linux_bpf
+
+package network
```

Comment:
Would be good to have this + `kprobes.go` in `ebpf`. It's a bit messy as-is and I can see us working to generalizing how we handle which kprobes exist + are enabled/disabled.

Reply:
yeah, I agree 100% they should both land at `ebpf` at some point.
right now moving `types.go` would be easy, but I'm not sure if I can move `kprobes.go` as it is. I think I would have to change the signature of 
```
EnabledKProbes(c *network.Config, pre410Knernel bool)` to something like:
```
EnabledKProbes(collectTCP, collectUDP, bpfDebug, pre410Knernel bool)
````
which is awful, but maybe better than trying to come up with a config interface interface for now? obviously in the future this function should take a single argument, but for now I'm just trying to move the code around. what are your thoughts on that?

Reply:
TBH I think it's best if I move `tracer.go` to `ebpf` and rename it to `NetworkTracer` for now.